### ISSUE TITLE
Beta 8

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Dragon-Born/google-api-python-client#107]
```
Version: Beta 8

Expected freeze date: Thrs Feb 23

Expected release date: Fri Feb 24
```

Original issue reported on code.google.com by `jcgregorio@google.com` on 21 Feb 2012 at 9:04

**Comment by GoogleCodeExporter** [comment]
```
Add release notes about body= parameters going away.
```

Original comment by `jcgregorio@google.com` on 23 Feb 2012 at 5:07

**Comment by GoogleCodeExporter** [comment]
```
v1.0beta8
 - Updated meda upload support.
 - Many fixes for batch requests.
 - Better handling for requests that don't require a body.
 - Fix issues with Google App Engine Python 2.7 runtime.
 - Better support for proxies.
 - All Storages now have a .delete() method.
 - Important changes which might break your code:
    * apiclient.anyjson has moved to oauth2client.anyjson.
    * Some calls, for example, taskqueue().lease() used to require a parameter
      named body. In this new release only methods that really need to send a body
      require a body parameter, and so you may get errors about an unknown
      'body' parameter in your call. The solution is to remove the unneeded
      body={} parameter.

```

Original comment by `jcgregorio@google.com` on 2 Mar 2012 at 9:45
- Changed state: **Fixed**
